src/main/assets/activities.json

```diff
@@ -1,5 +1,19 @@
 [
   {
+    "name": "Material Design List",
+    "description": "This example creates a example using a list with a new AppBar and FabButton of Android Support Design Library",
```

Comment:
"This example creates a example..."

What about something like:
"This example shows how to use a list with a the AppBar and FabButton from the Android Support Design Library"